config/locales/en.yml

```diff
@@ -141,7 +141,7 @@ en:
     days_passed:
       one: 1 day
       other: "%{count} days"
-      last: "%{count} or more days" 
+      last: "%{count} or more days"
```

Comment:
Ah, I was looking for how to do this the other day. Nice!

Reply:
Yep I have a vim script that removed whitespace on saving files - quite useful

Reply:
Oh, lol I meant the other, last syntax - I didn't realise that wasn't what you changed here 😂